src/HotChocolate/Core/src/Types.Mutations/MutationContextData.cs

```diff
@@ -10,8 +10,8 @@ internal class MutationContextData
         ObjectFieldDefinition definition,
         string? inputTypeName,
         string? inputArgumentName,
-        string? payloadFieldName,
         string? payloadTypeName,
+        string? payloadFieldName,
```

Comment:
can you also swap the initialization?

 PayloadFieldName = payloadFieldName;
        PayloadTypeName = payloadTypeName;